flow/designs/asap7/ibex/config.mk

```diff
@@ -6,10 +6,10 @@ export DESIGN_NAME            = ibex_core
 export VERILOG_FILES         = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
 export SDC_FILE              = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
-export CORE_UTILIZATION       = 25 
+export CORE_UTILIZATION       =  40
 export CORE_ASPECT_RATIO      = 1
 export CORE_MARGIN            = 2
-export PLACE_DENSITY          = 0.60
+export PLACE_DENSITY_LB_ADDON  = 0.20
```

Comment:
Is PLACE_DENSITY_LB_ADDON for the autotuner rather than regular usage?

Reply:
Explicit setting of PLACE_DENSITY is not ideal as the user will not know what should the right value be.  and it is a function of the initial Core Utilization.  Ideally, this value should be determined dynamically  by the global placement with routability/timing  considerations. The  use of LB_ADD_ON at least makes it a function of the initial utilization so it easier to manage when we are ratcheting the design utilization.